python/metric_pipeline_tasks/MatchedCatalogsBase.py

```diff
@@ -77,3 +77,45 @@ def runQuantum(self, butlerQC,
         inputs['box'] = patch_box
         outputs = self.run(**inputs)
         butlerQC.put(outputs, outputRefs)
+
+
+class MatchedTractBaseTask(pipeBase.PipelineTask):
```

Comment:
It seems like one of MatchedCatTractBaseTask / MatchedCatalogTractBaseTask / MatchedCatalogsTractBaseTask would be more consistent with the overall naming scheme of classes.

Reply:
I matched what was there for MatchedBaseTask, but I agree that it would be better to adopt MatchedCatBaseTask and MatchedCatTractBaseTask (or similar) in the future.